tests/PuchasesClient.test.ts

```diff
@@ -0,0 +1,139 @@
+import { PurchasesClient } from '../src/clients/PuchasesClient';
+import td from 'testdouble';
+import { expect } from 'chai';
+import {
+  ApplicationUsername, GetCustomerPurchasesResultData,
+  GetCustomersBulkInfoResult_data
+} from './dummy-data'
+
+
+describe('PurchasesClient', () => {
+
+  const createClient = () => new PurchasesClient({
+    protocol: 'http',
+    hostname: 'some-url-billing.com',
+    port: 80,
+    pathnamePrefix: '/v2/customers',
+    clientId: 'test',
+    secret: 'api_secret'
+  });
+
```

Comment:
So since the server is private .. and no one will be having its own server .. , do I need to add a configuration for URL change ?
or we keep it private ?

Reply:
Let's keep a configuration, so we can run tests against our staging or development servers. However don't mention it in the documentation, cf remarks above.